CHANGES.md

```diff
@@ -32,6 +32,11 @@ To be released.
 
 ### Behavioral changes
 
+  - (Libplanet.Net) Return the fixed races between `Swarm<T>`'s dual systems
```

Comment:
It seems the link for #1789 was missing.

Reply:
https://github.com/planetarium/libplanet/pull/1798/commits/f8775fc1d15efe1a0453adfb97561d185f466098